packages/widgetbook_core/lib/src/settings/features/knobs/color_knob.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook_core/src/settings/features/knobs/knob_property.dart';
+import 'package:widgetbook_core/src/settings/widgets/widgets.dart';
+
+enum ColorOptions {
+  hex(name: 'HEX'),
+  rgb(name: 'RGB (tbd)'),
+  hsl(name: 'HSL (tbd)'),
+  hsb(name: 'HSB (tbd)');
+
+  const ColorOptions({
+    required this.name,
+  });
+
+  final String name;
+}
+
+Color? colorTryParse(String value) {
+  final colorValue = int.tryParse(value, radix: 16);
+  if (colorValue == null) {
+    return null;
+  }
+
+  if (colorValue < Colors.black.withAlpha(0).value ||
+      colorValue > Colors.white.value) {
+    return null;
+  }
+
+  return Color(colorValue);
+}
+
+String? colorErrorText(String value) {
+  const errorText = 'Not a color';
+  final color = colorTryParse(value);
+  return color == null ? errorText : null;
+}
+
+class ColorKnob extends StatefulWidget {
```

Comment:
When I tested it, the color didn't update in the use case when changing the color knob value.